server/src/controller/apis/user.js

```diff
@@ -0,0 +1,9 @@
+
+import userAPI from '../../services/user/api';
+import { Router } from 'express';
+
+const router = Router();
+
+router.get('/health', userAPI.health);
```

Comment:
Any specific reason why the route is set to '/health' ? Could this be '/post', '/profile' to better reflect our project's requirements?

Reply:
This route will help us to check whether the user-specific APIs are working or not.
This will give us the status 200 if the users API are reachable.
eg:
```
'/health': To check the health of the most basic route.
'/api/v1/users/health': To check the health of the user route.

```

Now this thing would help us in debugging and writing tests as well.